tests/lib/rules/key-spacing.js

```diff
@@ -1286,5 +1498,110 @@ ruleTester.run("key-spacing", rule, {
             { message: "Missing space before value for key 'a'.", line: 1, column: 6, type: "Identifier" },
             { message: "Extra space after key 'c'.", line: 1, column: 20, type: "Identifier" }
         ]
+    },
+
+    // https://github.com/eslint/eslint/issues/5613
+    {
+        code: [
+            "({",
+            "    longName:1,",
+            "    small    :2,",
+            "    xs      : 3",
+            "})"
+        ].join("\n"),
+        options: [{
```

Comment:
Now that auto-fixing has been added, you'll want to go through the tests that have errors and add the `output` code as well.